test_files/aggregate.d

```diff
@@ -18,4 +19,5 @@ struct Bar
 }
 
 extern __gshared const(char)* q;
-extern __gshared int function (int, int) e;
\ No newline at end of file
+
+extern __gshared int function (int, int) e;
```

Comment:
As `Output` class is getting info about locations of translated cursors, it tries to keep the newlines. I believe that feature is a bit improved in the subsequent PRs.